src/SFA.DAS.Roatp.CourseManagement.Web/Models/ProviderCourseLocations/ProviderCourseLocationViewModel.cs

```diff
@@ -44,17 +26,14 @@ public string DeliveryOption()
                 Id = providerCourseLocation.Id,
                 LocationName = providerCourseLocation.LocationName,
                 LocationType = providerCourseLocation.LocationType,
-                HasDayReleaseDeliveryOption = providerCourseLocation.HasDayReleaseDeliveryOption,
-                HasBlockReleaseDeliveryOption = providerCourseLocation.HasBlockReleaseDeliveryOption,
-                RegionName = providerCourseLocation.RegionName
+                DeliveryMethod = new DeliveryMethodModel 
+                { 
+                    HasDayReleaseDeliveryOption = providerCourseLocation.HasDayReleaseDeliveryOption,
+                    HasBlockReleaseDeliveryOption = providerCourseLocation.HasBlockReleaseDeliveryOption
+                },
+                RegionName = providerCourseLocation.RegionName,
+                SubregionName = providerCourseLocation.SubregionName
```

Comment:
I can't find any coverage for RegionName and SubregionName, can you add it if appropriate?

Reply:
Added.